src/Aks/Aks/help/New-AzAksCluster.md

```diff
@@ -110,6 +111,21 @@ New-AzAksCluster -ResourceGroupName myResourceGroup -Name myAKSCluster -AutoScal
 
 ## PARAMETERS
 
+### -AadProfile
+The Azure Active Directory configuration.
```

Comment:
This is a complex type. Can we add a new example (or add to an exisiting one) to demonstrate how it's used?

Reply:
Examples are added for New-AzAksCluster and Set-AzAksCluster.